### ISSUE TITLE
Datetime widget: Mouse wheel scroll does not work with Firefox on Linux

### ISSUE THREAD

**Issue by klemens-u** [nextcloud/calendar#2003]
Extracting on of the issues of #2002:

Thanks for the information.

>> Scrolling per mousewheel does not work.
> Can you please create a small screencast of that? I'm not able to reproduce it in Firefox on macOS.
(or any other browser i have tried so far)
Interesting. I can confirm it works without problems on the same system with Chrome.

The problem was reproducible on a second system running Ubuntu 18.04 and the same Firefox version (72). Interestingly this system has a small screen, so the whole window has scrollbars, and the mouse wheel scrolls the content, not the hours/minutes. So it seems like a focus issue.

**Comment by georgehrke** [comment]
> Interestingly this system has a small screen, so the whole window has scrollbars, and the mouse wheel scrolls the content, 

That makes it sound like a side effect of #1815 to me.

**Comment by georgehrke** [comment]
Testing it on a smaller screen, i was able to reproduce it. When scrolling, it scrolls the window scrollbars, not the time.

Closing as duplicate of #1815.

**Comment by klemens-u** [comment]
Please note, that there are no scrollbars on my system with a very large screen. So the problem could easily have another cause.

**Comment by georgehrke** [comment]
Okay, can you please create a small screencast like i requested in #2002 then? 

Still not able to reproduce on a full-size firefox.

**Comment by georgehrke** [comment]
@tcitworld Are you able to reproduce this?

**Comment by raimund-schluessler** [comment]
I can reproduce this in the event-popover. It works when the mouse-pointer is over the top-most number/hour, but it does not scroll when hoverin the numbers below. Mouse-pointer over the scrollbar works as well.

I have no idea why, though.

**Comment by klemens-u** [comment]
> Okay, can you please create a small screencast like i requested in #2002 then?
Sorry, there is really nothing to see. It just does not scroll.

**Comment by georgehrke** [comment]
Well, if I'm not able to reproduce it, it's rather difficult for me to provide a fix.
@klemens-u Looking at your profile, you seem to do web-developing yourself. Any chance you can try to debug this yourself?

**Comment by raimund-schluessler** [comment]
For me it starts working when I replace `overflow-y: scroll` with `overflow: scroll` (no `-y`) for the `<ul class="mx-time-list">` which encloses the time-slots. @klemens-u Could you try if that helps in your case as well?

<img width="1123" alt="scollbar" src="https://user-images.githubusercontent.com/2496460/74683982-d068e380-51ca-11ea-9d13-1806c5ee2206.png">

**Comment by klemens-u** [comment]
> For me it starts working when I replace overflow-y: scroll with overflow: scroll (no -y) for the <ul class="mx-time-list"> which encloses the time-slots. @klemens-u Could you try if that helps in your case as well?

Yes, it works when setting to `overflow: scroll;`. Btw it is `overflow-y: auto;` and not `overflow-y: scroll;` in my case.

**Comment by raimund-schluessler** [comment]
You are right, the default is auto.

**Comment by georgehrke** [comment]
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/DatetimePicker/index.scss#L307

**Comment by joshtrichards** [comment]
Can any of you still reproduce this today? I certainly can't.

**Comment by SebastianKrupinski** [comment]
Closing this as per comments above this seems to be fixed
